### DISCUSSION TITLE
Help, Custom tool support file as param?

### DISCUSSION THREAD

**Discussion by Ima-Beast** [langgenius/dify#9322] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

 I would like to ask about a file parsing service I developed. The service accepts a file as a parameter. When publishing this as a custom tool, does it support file uploads? If it does support file uploads, how should I write the OpenAPI specification?

**Comment by zolgear** [comment]
See #10796 pull request

Discussions - https://github.com/langgenius/dify/discussions/10687#discussioncomment-11318023

> replace `multipart/form-data:` to `multipart/form-data; boundary=----WebKitFormBoundary<<random>>:`
